src/VirtoCommerce.Platform.Core/ChangeLog/ILastModifiedDateTime.cs

```diff
@@ -4,7 +4,7 @@ namespace VirtoCommerce.Platform.Core.ChangeLog
 {
     public interface ILastModifiedDateTime
     {
-        DateTimeOffset LastModified { get; }
-        void Reset();
+        public DateTimeOffset GetLastModified(string entityName = null);
```

Comment:
It is a breaking change in a public interface. Is it safe to do so with this one, or better to use another interface to preserve compatibility?

Reply:
Also `public` is not needed in the interface declaration

Reply:
I'm sure nobody uses this call directly.

Reply:
No, we do use `ILastModifiedDateTime.Reset()` method to invalidate storefront cache in Heineken modules. IMO, dedicated interface for scope-based modified dates will be safer.

Reply:
I just patched this (preserved compatibility)

Reply:
Adding methods to the existing interface makes its old implementations non-compliant. Could you add another interface with new methods?